internal/provider/asset/aws/ecs_compute/resources.go

```diff
@@ -320,5 +321,59 @@ func (r *Resource) Delete(ctx context.Context, req resource.DeleteRequest, resp
 }
 
 func (r *Resource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
-	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
+	// https://developer.hashicorp.com/terraform/plugin/framework/resources/import#multiple-attributes
+	// always found in the following format: "{organization_id},{environment_id},{asset_id}" for this request
+	positionalKeys := []string{"organization_id", "environment_id", "asset_id"}
```

Comment:
is it pretty difficult to have this in a helper function? this seems like the type of thing that might break on us at any point on the whim of hashicorp and updating everywhere would be annoying

Reply:
Terraform announced they were moving towards stability on the plugin with v0.14, then deprecated a bunch of stuff in v0.15 before removing it in v0.16. So they're definitely still at the "whim" phase despite their assurances changes would be minimal going forward.

Reply:
I can try to put it in a helper. This is a weird circular dependency file that we've autogenerated for the most part to avoid issues with import cycles. 

I'll try that now